### ISSUE TITLE
sample logrotate conf does not reflect split-user

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#845]
Originally reported of Google Code with ID 789.

```
What software version are you running? Please provide the output of "gnt-
cluster --version", "gnt-cluster version", and "hspace --version".

$ gnt-cluster --version
gnt-cluster (ganeti v2.9.5) 2.9.5

$ gnt-cluster version 
Software version: 2.9.5
Internode protocol: 2090000
Configuration format: 2090000
OS api version: 20
Export interface: 0
VCS version: v2.9.5

$ hspace --version
hspace (ganeti) version v2.9.5
compiled with ghc 7.6
running on linux x86_64

<b>What distribution are you using?</b>

$ cat /etc/*-release
LSB_VERSION="core-2.0-noarch:core-3.2-noarch:core-4.0-noarch:core-2.0-x86_64:core-3.2-x86_64:core-4.0-x86_64"
SUSE Linux Enterprise Server 11 (x86_64)
VERSION = 11
PATCHLEVEL = 3

<b>What steps will reproduce the problem?</b>
1. build ganeti with a split-user setup (--with-user-prefix= --with-group-prefix=)
2. install sample doc/examples/ganeti.logrotate
3. run logrotate

<b>What is the expected output? What do you see instead?</b>
Expected is that logfiles have proper owner/group/permissions. Instead new created logfiles are owned by root:root with mode 0640 -> Ganeti daemons are unable to log.

<b>Please provide any additional information below.</b>
Attached is a "works for me" logrotate conf. Probably this can be parameterized for the correct user/group names.

Thanks, Sascha.
```
Originally added on 2014-04-02 06:59:06 +0000 UTC.

Attachments: 
[logrotate.conf](https://storage.googleapis.com/google-code-attachments/ganeti/issue-789/comment-0/logrotate.conf)

**Comment by Ganeti-Issues-Migrator** [comment]
```
Have you tried simply omitting the line "create 640 root root"?
logrotate should then preserve the permissions.
```
Originally added on 2014-04-14 08:36:51 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi,

no I haven't tried to omitting the line "create 640 root root". I was not aware of this. Will test it and report back.

Thanks, Sascha.
```
Originally added on 2014-04-14 08:59:38 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi,

I've testet logrotation with omitting the line "create 640 root root". And yes it keeps owner/group/permissions.

Thanks for the hint. Sascha.
```
Originally added on 2014-04-14 12:46:21 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 266ea4232e2503c2de200f9d2d3cfe86aaddb8e3
Author: Jose A. Lopes <jabolopes@google.com>
Date:   Mon Apr 14 14:52:34 2014 +0200

    Sample logrotate conf breaks permissions with split users
    
    The sample logrotate configuration file
    (doc/examples/ganeti.logrotate) contains the following line
    
      create 640 root root
    
    which is fine if Ganeti if configured without split users.  However,
    this will change permissions and ownership of log files if Ganeti is
    configured with split users enabled.  By removing this line, logrotate
    will preserve the permissions of the logfiles when rotating the logs.
    
    This fixes issue 789.
    
    Signed-off-by: Jose A. Lopes <jabolopes@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>
```
Originally added on 2014-04-14 12:59:17 +0000 UTC.

Mentioned Issue with GoogleCode ID 789 has been migrated to GitHub with ID #845.
Added Labels: Type-Defect Priority-Low
Changed State: Fixed
Added to Milestone: Release2.9

**Comment by Ganeti-Issues-Migrator** [comment]
```
released as part of 2.11.6
```
Originally added on 2014-09-22 15:58:21 +0000 UTC.

Changed State: Released
